### ISSUE TITLE
Add more fundamental programs of JAVA

### ISSUE THREAD

**Issue by chandrakant100** [chandrakant100/JAVA_PROGRAMS#6]
1:Add more fundamental programs which covers the basic concepts.
2:Try to give proper indentation and comments

**Comment by hryxna** [comment]
Hello, @chandrakant100 would you like me to create a different folder for each program, or should I directly add them into the repo?

**Comment by chandrakant100** [comment]
> Hello, @chandrakant100 would you like me to create a different folder for each program, or should I directly add them into the repo?

yeah make another `folder` and commit the programs you want to add.
make folder assignment<number> where number must be the increment of previous assignment
